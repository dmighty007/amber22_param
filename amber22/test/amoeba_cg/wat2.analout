
     ######################################################################
   ##########################################################################
  ###                                                                      ###
 ###            TINKER  ---  Software Tools for Molecular Design            ###
 ##                                                                          ##
 ##                        Version 6.2  February 2013                        ##
 ##                                                                          ##
 ##               Copyright (c)  Jay William Ponder  1990-2013               ##
 ###                           All Rights Reserved                          ###
  ###                                                                      ###
   ##########################################################################
     ######################################################################


 Total Numbers of Atoms and Interactions :

 Atoms in System                         6
 Bond Stretches                          4
 Angle Bends                             2
 Urey-Bradley                            2
 Polarizable Multipoles                  6

 Atom Type Definition Parameters :

   Atom  Symbol  Type  Class  Atomic   Mass  Valence  Description

     1     O      247     43     8    15.999    2     AMOEBA Water O          
     2     H      248     44     1     1.008    1     AMOEBA Water H          
     3     H      248     44     1     1.008    1     AMOEBA Water H          
     4     H      248     44     1     1.008    1     AMOEBA Water H          
     5     O      247     43     8    15.999    2     AMOEBA Water O          
     6     H      248     44     1     1.008    1     AMOEBA Water H          

 Van der Waals Parameters :

          Atom Number       Radius   Epsilon   Rad 1-4   Eps 1-4   Reduction

     1        1           1.7025    0.1100
     2        2           1.3275    0.0135                          0.9100
     3        3           1.3275    0.0135                          0.9100
     4        4           1.3275    0.0135                          0.9100
     5        5           1.7025    0.1100
     6        6           1.3275    0.0135                          0.9100

 Bond Stretching Parameters :

          Atom Numbers                         KS       Length

     1        1     2                      556.850    0.9572
     2        1     3                      556.850    0.9572
     3        4     5                      556.850    0.9572
     4        5     6                      556.850    0.9572

 Angle Bending Parameters :

             Atom Numbers                      KB      Angle   Fold    Type

     1        2     1     3                 48.700   108.500
     2        4     5     6                 48.700   108.500

 Urey-Bradley Parameters :

          Atom Numbers                        KUB    Distance

     1        2     3                      -7.6000    1.5326
     2        4     6                      -7.6000    1.5326

 Atomic Multipole Parameters :

            Atom    Coordinate Frame Definition       Multipole Moments

     1        1       2      3          Bisector   -0.51966
                                                    0.00000  0.00000  0.14279
                                                    0.37928
                                                    0.00000 -0.41809
                                                    0.00000  0.00000  0.03881
     2        2       1      3          Z-then-X    0.25983
                                                   -0.03859  0.00000 -0.05818
                                                   -0.03673
                                                    0.00000 -0.10739
                                                   -0.00203  0.00000  0.14412
     3        3       1      2          Z-then-X    0.25983
                                                   -0.03859  0.00000 -0.05818
                                                   -0.03673
                                                    0.00000 -0.10739
                                                   -0.00203  0.00000  0.14412
     4        4       5      6          Z-then-X    0.25983
                                                   -0.03859  0.00000 -0.05818
                                                   -0.03673
                                                    0.00000 -0.10739
                                                   -0.00203  0.00000  0.14412
     5        5       4      6          Bisector   -0.51966
                                                    0.00000  0.00000  0.14279
                                                    0.37928
                                                    0.00000 -0.41809
                                                    0.00000  0.00000  0.03881
     6        6       5      4          Z-then-X    0.25983
                                                   -0.03859  0.00000 -0.05818
                                                   -0.03673
                                                    0.00000 -0.10739
                                                   -0.00203  0.00000  0.14412

 Dipole Polarizability Parameters :

          Atom Number         Alpha        Polarization Group

     1        1              0.8370          1     2     3
     2        2              0.4960          1     2     3
     3        3              0.4960          1     2     3
     4        4              0.4960          4     5     6
     5        5              0.8370          4     5     6
     6        6              0.4960          4     5     6

 Total Number of Pairwise Atomic Interactions :

 Number of 1-2 Pairs                     4
 Number of 1-3 Pairs                     2

 List of 1-2 Connected Atomic Interactions :

       1       2
       1       3
       4       5
       5       6

 List of 1-3 Connected Atomic Interactions :

       2       3
       4       6

 Total Potential Energy :                 -5.0375 Kcal/mole

 Energy Component Breakdown :           Kcal/mole      Interactions

 Bond Stretching                           0.0203              4
 Angle Bending                             0.0115              2
 Urey-Bradley                             -0.0054              2
 Van der Waals                             2.3482              9
 Atomic Multipoles                        -6.1240             21
 Polarization                             -1.2880             21

 Total Electric Charge :                  0.00000 Electrons

 Dipole Moment Magnitude :                  2.545 Debyes

 Dipole X,Y,Z-Components :                  2.124       1.339       0.414

 Quadrupole Moment Tensor :                 8.152      -2.540      -1.389
      (Buckinghams)                        -2.540      -7.359      -2.756
                                           -1.389      -2.756      -0.793

 Principal Axes Quadrupole :               -8.844       0.189       8.655

 Radius of Gyration :                       1.637 Angstroms

 Center of Mass Coordinates :           -0.822483    0.083892   -0.227549
 Euler Angles (Phi/Theta/Psi) :            31.190      -8.787     108.261

 Moments of Inertia and Principal Axes :

             Moments (amu Ang^2)          X-, Y- and Z-Components of Axes

                      2.308              0.845418    0.038176    0.532739
                     77.856              0.511796   -0.343171   -0.787590
                     77.979              0.152754    0.938497   -0.309661

 Internal Virial Tensor :                  -0.379      -0.027       0.012
                                           -0.027      -0.261      -0.200
                                            0.012      -0.200       0.051

 dE/dV (Virial-based) :                 -0.000006 Kcal/mole/A**3
 dE/dV (Finite Diff) :                  -0.000006 Kcal/mole/A**3

 Pressure (Analytical, 0 K) :               0.411 Atmospheres
 Pressure (Numerical, 0 K) :                0.411 Atmospheres

 Potential Energy Breakdown over Atoms :

  Atom        EB          EA          EBA         EUB         EAA         EOPB
              EOPD        EID         EIT         ET          EPT         EBT
              ETT         EV          EC          ECD         ED          EM
              EP          ER          ES          ELF         EG          EX

     1      0.0031      0.0034      0.0000      0.0000      0.0000      0.0000
            0.0000      0.0000      0.0000      0.0000      0.0000      0.0000
            0.0000      1.2129      0.0000      0.0000      0.0000     -6.4472
           -0.8455      0.0000      0.0000      0.0000      0.0000      0.0000

     2      0.0016      0.0000      0.0000     -0.0016      0.0000      0.0000
            0.0000      0.0000      0.0000      0.0000      0.0000      0.0000
            0.0000     -0.0194      0.0000      0.0000      0.0000      0.4107
           -0.0067      0.0000      0.0000      0.0000      0.0000      0.0000

     3      0.0015      0.0000      0.0000     -0.0016      0.0000      0.0000
            0.0000      0.0000      0.0000      0.0000      0.0000      0.0000
            0.0000     -0.0194      0.0000      0.0000      0.0000      0.4084
           -0.0070      0.0000      0.0000      0.0000      0.0000      0.0000

     4      0.0070      0.0000      0.0000     -0.0012      0.0000      0.0000
            0.0000      0.0000      0.0000      0.0000      0.0000      0.0000
            0.0000      1.1285      0.0000      0.0000      0.0000     -2.5175
           -0.7505      0.0000      0.0000      0.0000      0.0000      0.0000

     5      0.0071      0.0081      0.0000      0.0000      0.0000      0.0000
            0.0000      0.0000      0.0000      0.0000      0.0000      0.0000
            0.0000      0.0611      0.0000      0.0000      0.0000     -3.1210
           -0.2183      0.0000      0.0000      0.0000      0.0000      0.0000

     6      0.0001      0.0000      0.0000     -0.0012      0.0000      0.0000
            0.0000      0.0000      0.0000      0.0000      0.0000      0.0000
            0.0000     -0.0155      0.0000      0.0000      0.0000      0.0017
            0.1357      0.0000      0.0000      0.0000      0.0000      0.0000
