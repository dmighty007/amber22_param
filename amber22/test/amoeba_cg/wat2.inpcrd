%VERSION  VERSION_STAMP = V0001.000  DATE = 09/09/13  07:54:20 
%FLAG TITLE
%FORMAT(a)
water dimer
%FLAG ATOMIC_COORDS_SIMULATION_TIME
%FORMAT(E16.8)
  0.00000000E+00
%FLAG ATOMIC_COORDS_NUM_LIST
%FORMAT(i8)
       6
%FLAG ATOMIC_COORDS_LIST
%COMMENT   dimension = (3,6)           
%FORMAT(3e20.12)
  0.337466000000E+00  0.854015000000E+00  0.783700000000E-02
  0.891670000000E+00  0.385821000000E+00  0.635868000000E+00
  0.827103000000E+00  0.941462000000E+00 -0.812736000000E+00
 -0.129863000000E+01 -0.135260000000E+00 -0.289956000000E+00
 -0.203703800000E+01 -0.723875000000E+00 -0.474884000000E+00
 -0.284337200000E+01 -0.258983000000E+00 -0.253714000000E+00
%FLAG UNIT_CELL_PARAMETERS
%COMMENT lengths a,b,c; then angles alpha,beta,gamma
%FORMAT(3e20.12)
  0.320000000000E+02  0.320000000000E+02  0.320000000000E+02
  0.900000000000E+02  0.900000000000E+02  0.900000000000E+02
