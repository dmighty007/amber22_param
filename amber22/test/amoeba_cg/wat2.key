#
#  Keyfile for water dimer test
#

parameters            /home/case/mm/tinker/params/amoebapro13.prm
#verbose
randomseed            123456789
neighbor-list

#
#  Define the Periodic Box and Cutoffs
#

a-axis                32.0
vdw-cutoff            12.0

#
#  Set Parameters for Ewald Summation
#

ewald
ewald-cutoff          7.0
pme-grid              32 32 32
fft-package           fftw
polar-eps             0.00001

