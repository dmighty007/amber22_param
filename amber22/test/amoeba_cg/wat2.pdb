HEADER
COMPND
SOURCE
HETATM    1  O   HOH     1       0.337   0.854   0.008
HETATM    2  H   HOH     1       0.892   0.386   0.636
HETATM    3  H   HOH     1       0.827   0.941  -0.813
HETATM    4  H   HOH     2      -1.299  -0.135  -0.290
HETATM    5  O   HOH     2      -2.037  -0.724  -0.475
HETATM    6  H   HOH     2      -2.843  -0.259  -0.254
CONECT    1    2    3
CONECT    2    1
CONECT    3    1
CONECT    4    5
CONECT    5    4    6
CONECT    6    5
END
