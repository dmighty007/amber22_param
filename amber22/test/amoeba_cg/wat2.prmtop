%VERSION  VERSION_STAMP = V0001.000  DATE = 09/09/13  07:54:20
%FLAG TITLE
%FORMAT(a)
water dimer
%FLAG POINTERS
%FORMAT(10I8)
       6       1       1       1       1       1       1       1       0       0
       6       2       1       1       1       1       1       1       1       1
       0       0       0       0       0       0       0       1       0       0
       0       0
%FLAG ATOM_NAME
%FORMAT(20a4)
O   H   H   H   O   H   
%FLAG CHARGE
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00
%FLAG MASS
%FORMAT(5E16.8)                                                                 
  0.15999000E+02  0.10080000E+01  0.10080000E+01  0.10080000E+01  0.15999000E+02
  0.10080000E+01
%FLAG ATOM_TYPE_INDEX
%FORMAT(10I8)
       1       1       1       1       1       1
%FLAG NUMBER_EXCLUDED_ATOMS
%FORMAT(10I8)
       2       1       0       2       1       0
%FLAG NONBONDED_PARM_INDEX
%FORMAT(10I8)
       1
%FLAG RESIDUE_LABEL
%FORMAT(20a4)
HOH HOH 
%FLAG RESIDUE_POINTER
%FORMAT(10I8)
       1       4
%FLAG BOND_FORCE_CONSTANT
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG BOND_EQUIL_VALUE
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG ANGLE_FORCE_CONSTANT
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG ANGLE_EQUIL_VALUE
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG DIHEDRAL_FORCE_CONSTANT
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG DIHEDRAL_PERIODICITY
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG DIHEDRAL_PHASE
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG SOLTY
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG LENNARD_JONES_ACOEF
%FORMAT(5E16.8)                                                                 
  0.10000000E+01
%FLAG LENNARD_JONES_BCOEF
%FORMAT(5E16.8)                                                                 
  0.10000000E+01
%FLAG BONDS_INC_HYDROGEN
%FORMAT(10I8)
       1       1       1
%FLAG BONDS_WITHOUT_HYDROGEN
%FORMAT(10I8)
       1       1       1
%FLAG ANGLES_INC_HYDROGEN
%FORMAT(10I8)
       1       1       1       1
%FLAG ANGLES_WITHOUT_HYDROGEN
%FORMAT(10I8)
       1       1       1       1
%FLAG DIHEDRALS_INC_HYDROGEN
%FORMAT(10I8)
       1       1       1       1       1
%FLAG DIHEDRALS_WITHOUT_HYDROGEN
%FORMAT(10I8)
       1       1       1       1       1
%FLAG EXCLUDED_ATOMS_LIST
%FORMAT(10I8)
       2       3       3       5       6       6
%FLAG HBOND_ACOEF
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG HBOND_BCOEF
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG HBCUT
%FORMAT(5E16.8)                                                                 
  0.10000000E+01
%FLAG AMBER_ATOM_TYPE
%FORMAT(20a4)
O   H   H   H   O   H   
%FLAG TREE_CHAIN_CLASSIFICATION
%FORMAT(20a4)
BLA BLA BLA BLA BLA BLA 
%FLAG JOIN_ARRAY
%FORMAT(10I8)
       0       0       0       0       0       0
%FLAG IROTAT
%FORMAT(10I8)
       0       0       0       0       0       0
%FLAG SOLVENT_POINTERS
%FORMAT(10I8)
       0       2       0
%FLAG ATOMS_PER_MOLECULE
%FORMAT(10I8)
       3       3
%FLAG BOX_DIMENSIONS
%FORMAT(5E16.8)                                                                 
  0.90000000E+02  0.10000000E+01  0.10000000E+01  0.10000000E+01
%FLAG AMOEBA_FORCEFIELD
%COMMENT This indicates that this parm file is specific to amoeba
%COMMENT This must be present if do_amoeba(in mdin) is 1
%COMMENT This must NOT be present if do_amoeba is 0
%FORMAT(i5)
    1
%FLAG AMOEBA_ATOM_TYPE_INDEX
%COMMENT   dimension = (6)             
%FORMAT(10I8)
     247     248     248     248     247     248
%FLAG AMOEBA_ATOMIC_NUMBER
%COMMENT   dimension = (6)             
%FORMAT(10I8)
       8       1       1       1       8       1
%FLAG AMOEBA_ATOM_CLASS_INDEX
%COMMENT   dimension = (6)             
%FORMAT(10I8)
      43      44      44      44      43      44
%FLAG AMOEBA_REGULAR_BOND_NUM_LIST
%FORMAT(I8)
       4
%FLAG AMOEBA_REGULAR_BOND_LIST
%COMMENT   dimension = (3,4)           
%FORMAT(10I8)
       1       2       1       1       3       1       4       5       1       5
       6       1
%FLAG AMOEBA_REGULAR_BOND_NUM_PARAMS
%FORMAT(I8)
       1
%FLAG AMOEBA_REGULAR_BOND_FORCE_CONSTANT
%COMMENT   dimension = (1)             
%FORMAT(5E16.8)                                                                 
  0.55685000E+03
%FLAG AMOEBA_REGULAR_BOND_EQUIL_VALUE
%COMMENT   dimension = (1)             
%FORMAT(5E16.8)                                                                 
  0.95720000E+00
%FLAG AMOEBA_REGULAR_BOND_FTAB_DEGREE
%FORMAT(I8)
       4
%FLAG AMOEBA_REGULAR_BOND_FTAB_COEFFS
%COMMENT   dimension = (0:4)           
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  0.10000000E+01 -0.25500000E+01  0.37931250E+01
%FLAG AMOEBA_UREY_BRADLEY_BOND_NUM_LIST
%FORMAT(I8)
       2
%FLAG AMOEBA_UREY_BRADLEY_BOND_LIST
%COMMENT   dimension = (3,2)           
%FORMAT(10I8)
       2       3       1       4       6       1
%FLAG AMOEBA_UREY_BRADLEY_BOND_NUM_PARAMS
%FORMAT(I8)
       1
%FLAG AMOEBA_UREY_BRADLEY_BOND_FORCE_CONSTANT
%COMMENT   dimension = (1)             
%FORMAT(5E16.8)                                                                 
 -0.76000000E+01
%FLAG AMOEBA_UREY_BRADLEY_BOND_EQUIL_VALUE
%COMMENT   dimension = (1)             
%FORMAT(5E16.8)                                                                 
  0.15326000E+01
%FLAG AMOEBA_UREY_BRADLEY_BOND_FTAB_DEGREE
%FORMAT(I8)
       2
%FLAG AMOEBA_UREY_BRADLEY_BOND_FTAB_COEFFS
%COMMENT   dimension = (0:2)           
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  0.10000000E+01
%FLAG AMOEBA_REGULAR_ANGLE_NUM_LIST
%FORMAT(I8)
       2
%FLAG AMOEBA_REGULAR_ANGLE_LIST
%COMMENT   dimension = (4,2)           
%FORMAT(10I8)
       2       1       3       1       4       5       6       1
%FLAG AMOEBA_REGULAR_ANGLE_NUM_PARAMS
%FORMAT(I8)
       1
%FLAG AMOEBA_REGULAR_ANGLE_FORCE_CONSTANT
%COMMENT   dimension = (1)             
%FORMAT(5E16.8)                                                                 
  0.48700000E+02
%FLAG AMOEBA_REGULAR_ANGLE_EQUIL_VALUE
%COMMENT   dimension = (1)             
%FORMAT(5E16.8)                                                                 
  0.10850000E+03
%FLAG AMOEBA_REGULAR_ANGLE_FTAB_DEGREE
%FORMAT(I8)
       6
%FLAG AMOEBA_REGULAR_ANGLE_FTAB_COEFFS
%COMMENT   dimension = (0:6)           
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  0.10000000E+01 -0.14000000E-01  0.56000000E-04
 -0.70000000E-06  0.22000000E-07
%FLAG AMOEBA_VDW_ATOM_TYPES_NUM_LIST
%FORMAT(I8)
       6
%FLAG AMOEBA_VDW_ATOM_TYPES_LIST
%COMMENT   dimension = (1,6)               
%FORMAT(10I8)
       1       2       2       2       1       2
%FLAG AMOEBA_VDW_ATOM_PARENT_LIST
%COMMENT   dimension = (1,6)               
%FORMAT(10I8)
       1       1       1       5       5       5
%FLAG AMOEBA_VDW_PARENT_COORD_WEIGHT_LIST
%COMMENT   dimension = (1,6)               
%FORMAT(5E16.8)                                                                 
  0.10000000E+01  0.90000000E-01  0.90000000E-01  0.90000000E-01  0.10000000E+01
  0.90000000E-01
%FLAG AMOEBA_VDW_BUFFER_DELTA
%FORMAT(E16.8)
  0.70000000E-01
%FLAG AMOEBA_VDW_BUFFER_GAMMA
%FORMAT(E16.8)
  0.12000000E+00
%FLAG AMOEBA_VDW_PARAMS_NUM_LIST
%FORMAT(I8)
       2
%FLAG AMOEBA_VDW_MIXED_RADII_LIST
%COMMENT   dimension = (2,2)               
%FORMAT(5E16.8)                                                                 
  0.34050000E+01  0.31214215E+01  0.31214215E+01  0.26550000E+01
%FLAG AMOEBA_VDW_MIXED_EPSILONS_LIST
%COMMENT   dimension = (2,2)               
%FORMAT(5E16.8)                                                                 
  0.11000000E+00  0.29615390E-01  0.29615390E-01  0.13500000E-01
%FLAG AMOEBA_LOCAL_FRAME_MULTIPOLES_NUM_LIST
%FORMAT(I8)
       6
%FLAG AMOEBA_LOCAL_FRAME_MULTIPOLES_LIST
%COMMENT   dimension = (10,6)           
%FORMAT(5E16.8)                                                                 
 -0.51966000E+00  0.00000000E+00  0.00000000E+00  0.14279000E+00  0.18964000E+00
 -0.20904500E+00  0.19405000E-01  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.25983000E+00 -0.38590000E-01  0.00000000E+00 -0.58180000E-01 -0.18365000E-01
 -0.53695000E-01  0.72060000E-01  0.00000000E+00 -0.20300000E-02  0.00000000E+00
  0.25983000E+00 -0.38590000E-01  0.00000000E+00 -0.58180000E-01 -0.18365000E-01
 -0.53695000E-01  0.72060000E-01  0.00000000E+00 -0.20300000E-02  0.00000000E+00
  0.25983000E+00 -0.38590000E-01  0.00000000E+00 -0.58180000E-01 -0.18365000E-01
 -0.53695000E-01  0.72060000E-01  0.00000000E+00 -0.20300000E-02  0.00000000E+00
 -0.51966000E+00  0.00000000E+00  0.00000000E+00  0.14279000E+00  0.18964000E+00
 -0.20904500E+00  0.19405000E-01  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.25983000E+00 -0.38590000E-01  0.00000000E+00 -0.58180000E-01 -0.18365000E-01
 -0.53695000E-01  0.72060000E-01  0.00000000E+00 -0.20300000E-02  0.00000000E+00
%FLAG AMOEBA_FRAME_DEF_NUM_LIST
%FORMAT(I8)
      14
%FLAG AMOEBA_FRAME_DEF_LIST
%COMMENT   dimension = (5,14)           
%FORMAT(10I8)
       1       1       1       2       2       1       1       1       3       2
       1       2       1       3       1       2       1       2       1       1
       2       2       2       3       1       3       1       3       1       1
       3       2       3       2       1       4       1       4       5       1
       4       2       4       6       1       5       1       5       4       2
       5       1       5       6       2       5       2       5       6       1
       6       1       6       5       1       6       2       6       4       1
%FLAG AMOEBA_ADJUST_NUM_LIST
%FORMAT(I8)
       6
%FLAG AMOEBA_ADJUST_LIST
%COMMENT   dimension = (3,6)            
%FORMAT(10I8)
       1       2       5       1       3       5       2       3       6       4
       5       5       4       6       6       5       6       5
%FLAG AMOEBA_ADJUST_VDW_WEIGHTS_LIST
%COMMENT   dimension = (9)
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  0.10000000E+01  0.10000000E+01  0.00000000E+00
  0.00000000E+00  0.10000000E+01  0.10000000E+01  0.10000000E+01
%FLAG AMOEBA_ADJUST_MPOLE_WEIGHTS_LIST
%COMMENT   dimension = (9)
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  0.40000000E+00  0.80000000E+00  0.00000000E+00
  0.00000000E+00  0.40000000E+00  0.80000000E+00  0.10000000E+01
%FLAG AMOEBA_ADJUST_DIRECT_WEIGHTS_LIST
%COMMENT   dimension = (9)
%FORMAT(5E16.8)                                                                 
  0.10000000E+01  0.10000000E+01  0.10000000E+01  0.10000000E+01  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
%FLAG AMOEBA_ADJUST_POLAR_WEIGHTS_LIST
%COMMENT   dimension = (9)
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  0.10000000E+01  0.10000000E+01  0.00000000E+00
  0.00000000E+00  0.50000000E+00  0.10000000E+01  0.10000000E+01
%FLAG AMOEBA_ADJUST_MUTUAL_WEIGHTS_LIST
%COMMENT   dimension = (9)
%FORMAT(5E16.8)                                                                 
  0.10000000E+01  0.10000000E+01  0.10000000E+01  0.10000000E+01  0.10000000E+01
  0.10000000E+01  0.10000000E+01  0.10000000E+01  0.10000000E+01
%FLAG AMOEBA_POLARIZABILITY_NUM_LIST
%FORMAT(I8)
       6
%FLAG AMOEBA_POLARIZABILITY_LIST
%COMMENT   dimension = (6)              
%FORMAT(5E16.8)                                                                 
  0.83700000E+00  0.49600000E+00  0.49600000E+00  0.49600000E+00  0.83700000E+00
  0.49600000E+00
