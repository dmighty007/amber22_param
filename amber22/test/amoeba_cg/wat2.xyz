     6
     1  O      0.337466    0.854015    0.007837   247     2     3
     2  H      0.891670    0.385821    0.635868   248     1
     3  H      0.827103    0.941462   -0.812736   248     1
     4  H     -1.298630   -0.135260   -0.289956   248     5
     5  O     -2.037038   -0.723875   -0.474884   247     4     6
     6  H     -2.843372   -0.258983   -0.253714   248     5
