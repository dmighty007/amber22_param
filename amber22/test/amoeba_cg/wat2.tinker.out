
     ######################################################################
   ##########################################################################
  ###                                                                      ###
 ###            TINKER  ---  Software Tools for Molecular Design            ###
 ##                                                                          ##
 ##                        Version 6.2  February 2013                        ##
 ##                                                                          ##
 ##               Copyright (c)  Jay William Ponder  1990-2013               ##
 ###                           All Rights Reserved                          ###
  ###                                                                      ###
   ##########################################################################
     ######################################################################


 Limited Memory BFGS Quasi-Newton Optimization :

 QN Iter     F Value      G RMS     F Move    X Move   Angle  FG Call  Comment

     0       -5.0375     0.0076                                   1

 LBFGS  --  Normal Termination due to SmallGrad

 Final Function Value :           -5.0375
 Final RMS Gradient :              0.0076
 Final Gradient Norm :             0.0185
